backend/pkg/serve_options.go

```diff
@@ -19,5 +19,5 @@ func getServeOptions() datasource.ServeOpts {
 		QueryDataHandler:    sqlDatasource,
 		CheckHealthHandler:  sqlDatasource,
 		CallResourceHandler: httpHandler,
-	}
+	}, sqlDatasource
```

Comment:
Wondering if there's any redundancy to using multiple returns here (since `QueryDataHandler` etc. are already assigned the data source)?

Reply:
Yeah but i cant seem to access the sql data source hehe